### PR TITLE
test: check matthew

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by chanonroy — APPROVED** [review]
Great addition! You have a bright future ahead of you!

File: src/index.test.ts [hunk 1]
```diff
@@ -31,3 +31,7 @@ test('is FALSE with number', () => {
 test('is Chanon equal to Hussein', () => {
   expect(isChanon('Hussein')).toBe(false)
 })
+
+test('is Chanon equal to matthew', () => {
+  expect(isChanon('matthew')).toBe(false)
+})
```
Comment (husshegz) [inline src/index.test.ts:37]: Did we consider the capitalized version for this Hebrew-Biblical name ? how about the latin spelling ? 🤔

**Comment by husshegz** [comment]
@mattietea Can you look into this ?

**Comment by mattietea** [comment]
> @mattietea Can you look into this ?

Oof, good shout. Created an [issue here](https://github.com/chanonroy/is-chanon/issues/64). Let's setup a meeting to comb through the backlog and prioritise ASAP. cc. @chanonroy

**Comment by chanonroy** [comment]
Thank you, gentleman.

**Pull request merged**